js/26_09_19.test.js

```diff
@@ -0,0 +1,13 @@
+const likes = require('./26_09_19');
+
+
+describe('check likes', () => {
```

Comment:
You could name the describe block the way it is easy to find the code that it tests :slightly_smiling_face: Think of a way how to find the file name which the `check likes` describe block tests.

The file's name is a date. Maybe try adding the date here?

Reply:
I'm totally agree. I've made change in block name.